test/integration/server/inspec/server_spec.rb

```diff
@@ -26,8 +22,8 @@
 end
 
 %w(mysqld_safe mysqld).each do |p|
-  describe process(p) do
-    it { should be_running }
+  describe processes(p) do
+    its('states') { should eq ['R<'] }
```

Comment:
I think for our purposes this is good enough:

```suggestion
    it { should exist }
```